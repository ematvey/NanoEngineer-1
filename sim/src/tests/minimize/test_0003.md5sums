stdout 635EAD435B36BDC5849EBA59B7252E5F
stderr D41D8CD98F00B204E9800998ECF8427E
