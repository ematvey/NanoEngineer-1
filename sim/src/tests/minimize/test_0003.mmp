mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (10.000000) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (-0.536041, -0.061509, 0.303863, 0.785203) (5.149980) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (1)
mol (Chunk-C93) def
atom 1 (6) (-247, 1824, -328) def
atom 2 (0) (-705, 1410, -819) def
bond1 1
atom 3 (0) (-637, 2430, -7) def
bond1 1
atom 4 (6) (-584, 2236, 1718) def
bond1 1
atom 5 (0) (-701, 1934, 2436) def
bond1 4
atom 6 (0) (-732, 3010, 1695) def
bond1 4
atom 7 (6) (265, 475, 567) def
bond1 4
atom 8 (0) (-354, 415, 85) def
bond1 7
atom 9 (0) (925, 196, 237) def
bond1 7
atom 10 (6) (1671, 1055, -2062) def
bond1 1
atom 11 (0) (2261, 1134, -2555) def
bond1 10
atom 12 (6) (1882, 322, -80) def
bond1 10
atom 13 (0) (2000, 614, 625) def
bond1 12
atom 14 (6) (1066, -860, 449) def
bond1 7 12
atom 15 (0) (1395, -1208, 1074) def
bond1 14
atom 16 (0) (733, -1384, -34) def
bond1 14
atom 17 (6) (-1397, -20, -2263) def
bond1 10
atom 18 (0) (-1719, -606, -1847) def
bond1 17
atom 19 (0) (-1936, 388, -2668) def
bond1 17
mol (Chunk-C114) def
atom 20 (6) (1862, 228, -2527) def
bond1 17
atom 21 (0) (2225, 926, -2496) def
bond1 20
atom 22 (0) (2203, -259, -3043) def
bond1 20
atom 23 (6) (2775, -212, -1199) def
bond1 20 12
atom 24 (0) (3473, 149, -1218) def
bond1 23
atom 25 (0) (2844, -989, -1094) def
bond1 23
egroup (1)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part 1
