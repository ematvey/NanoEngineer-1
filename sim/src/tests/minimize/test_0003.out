======= test_0003.test =======
TYPE fail
PROGRAM simulator -f300 -m -X -x test_0003.mmp
